palantirnet/drupal-skeleton
# Test removing contrib modules from composer.json

### Description

Remove all of the contrib modules from the skeleton since they are installed by The Build.

### Testing instructions

1. Create a new project using this branch:
`composer create-project palantirnet/drupal-skeleton example dev-composer-remove-contrib --no-interaction
2. Verify the desired contrib modules are properly installed.

### Remaining tasks

- [ ] Revert the addition of the `.gitkeep` file and create a new release of the build with a step to create the `modules/contrib` directory during `drupal-first-install`.
- [ ] Revert change to the-build version once https://github.com/palantirnet/the-build/pull/225 is in a release.
